### PR TITLE
Migrate to new sequence model

### PR DESCRIPTION
**Pull request by rpiaggio**

### CODE REVIEW

File: explore/src/main/scala/explore/config/sequence/GmosSequenceRow.scala [hunk 1]
```diff
@@ -35,14 +37,14 @@ sealed trait GmosSequenceRow:
   // TODO Not in model yet, we are just simulating
   lazy val guided: Boolean                                  =
```
Comment (cquiroz) [inline explore/src/main/scala/explore/config/sequence/GmosSequenceRow.scala:38]: Is this whether offsets are guided?

**Review by cquiroz — APPROVED** [review]
LGTM

File: explore/src/main/scala/explore/config/sequence/GmosSequenceRow.scala (hunk 1 above)
Comment (rpiaggio) [reply explore/src/main/scala/explore/config/sequence/GmosSequenceRow.scala:38]: I'm assuming so. I took the columns a while ago from the inception document.

Comment (rpiaggio) [reply explore/src/main/scala/explore/config/sequence/GmosSequenceRow.scala:38]: Great, I saw that, it's actually in the query now and in memory. We just have to switch to showing it.

**Pull request merged**